### ISSUE TITLE
Skipping a platform when building wheels

### ISSUE THREAD

**Issue by nden** [spacetelescope/action-publish_to_pypi#18]
Is there a way I can skip a platform when building wheels for a specific repository?
More specifically, is there a way to pass a configuration option to cibuildwheel skipping a platform, e.g.
```
CIBW_SKIP: "*-manylinux_i686"
```

**Comment by zanecodes** [comment]
Yes, you should be able to configure this in your `.github/workflows/publish-to-pypi.yml` by adding [an `env` key](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv) to [the `publish:` key](https://github.com/zanecodes/.github/blob/reusable/workflow-templates/publish-to-pypi.yml#L8):

```
...

jobs:
  publish:
    ...
    env:
      CIBW_SKIP: *-manylinux_i686
```

This _should_ get passed along to the steps in the workflow, although I haven't tested this myself.
